src/commands/deployImage/imageSource/containerRegistry/acr/createAcr/RegistryCreateStep.ts

```diff
@@ -30,6 +33,28 @@ export class RegistryCreateStep extends AzureWizardExecuteStep<CreateAcrContext>
     }
 
     public shouldExecute(context: CreateAcrContext): boolean {
-        return !!context.newRegistryName && !!context.newRegistrySku;
+        return !context.registry;
+    }
+
+    protected initSuccessOutput(context: CreateAcrContext): ExecuteActivityOutput {
+        return {
+            item: new GenericTreeItem(undefined, {
+                contextValue: createActivityChildContext(['registryCreateStep', activitySuccessContext]),
+                label: localize('createRegistryLabel', 'Create Azure Container Registry "{0}"', context.newRegistryName),
```

Comment:
I'd just say "Create registry". Bit weird that you have all of the other resources be generic, and this one is branded.